oneview_redfish_toolkit/services/zone_service.py

```diff
@@ -97,16 +97,35 @@ def get_zone_ids_by_templates(self, server_profile_templates):
                 server_profile_templates: the list of Server Profile Template
         """
         zone_ids = []
+        expected_enclosures_uris = self._get_enclosures_uris()
```

Comment:
To make it more clear, please rename to "enclosure_uris_with_drives" or "enclosure_uris_with_valid_drive_enclosures". The method also could be renamed to "_get_enclosures_uris_with_valid_drive_enclosures" for example.